src/OmniSharp.DotNet.ProjectModel/AnalyzerOptions.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
ℹ️ Per the MIT license, we need to keep this copyright header. I've also included the LICENSE.md from https://github.com/dotnet/cli in the root of OmniSharp.DotNet.ProjectModel.